### PR TITLE
hamid/80661/add-poa-into-signup-wizard

### PR DESCRIPTION
**Pull request by hamid-deriv**
## Changes:

Please include a summary of the change and which issue is fixed below:
-   ...

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.scss [hunk 1]
```diff
@@ -10,6 +10,16 @@
         &-hint {
             margin-top: 0.8rem;
         }
+        &-form-container {
+            margin: 0 auto;
+            margin-top: 3.2rem;
+            width: 422px;
```
Comment (george-usynin-binary) [inline packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.scss:16]:
```suggestion
            width: 42.2rem;
```

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.scss [hunk 2]
```diff
@@ -22,5 +32,31 @@
                 width: 422px;
```
Comment (george-usynin-binary) [inline packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.scss:32]:
```suggestion
                width: 42.2rem;
```

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.scss [hunk 3]
```diff
@@ -10,6 +10,16 @@
         &-hint {
             margin-top: 0.8rem;
         }
+        &-form-container {
+            margin: 0 auto;
+            margin-top: 3.2rem;
```
Comment (george-usynin-binary) [inline packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.scss:15]:
```suggestion
                margin: 0 auto 32px;
```

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.tsx [hunk 4]
```diff
@@ -77,6 +79,18 @@ const SignupWizard = ({ closeWizard }: TSignupWizardProps) => {
                         >
                             <Selfie selfie={steps_state.selfie} onSelect={setSelfie} />
                         </Wizard.Step>
+                        <Wizard.Step
+                            title='Address verification'
```
Comment (george-usynin-binary) [inline packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.tsx:83]:
```suggestion
                            title={localize('Address verification')}
```

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.tsx [hunk 5]
```diff
@@ -77,6 +79,18 @@ const SignupWizard = ({ closeWizard }: TSignupWizardProps) => {
...
+                            is_submit_disabled={steps_state.address_verification_disabled}
```
Comment (george-usynin-binary) [inline packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.tsx:84]: @hamid-deriv could we add is_ to address_verification_disabled property

Comment (hamid-deriv) [reply packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.tsx:84]: Sorted.

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.tsx (hunk 4 above)
Comment (hamid-deriv) [reply packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.tsx:83]: Sorted.

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.scss (hunk 3 above)
Comment (hamid-deriv) [reply packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.scss:15]: Fixed.

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/steps/address-verification/proof-of-address-form/proof-of-address-form.scss [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+        }
+
+        &-dashboard {
+            max-width: calc(100% - 42rem);
+        }
+
+        &-list {
+            width: 100%;
+            flex: 1;
+            margin-right: 16px;
```
Comment (george-usynin-binary) [inline packages/cashier/src/pages/payment-agent/payment-agent-signup/steps/address-verification/proof-of-address-form/proof-of-address-form.scss:17]:
```suggestion
            margin-right: 1.6rem;
```

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/steps/address-verification/proof-of-address-form/proof-of-address-form.scss [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+
+            @include mobile {
+                flex-direction: column;
+                margin-right: 0;
+            }
+        }
+
+        &-file {
+            flex: 1;
+            height: 240px;
```
Comment (george-usynin-binary) [inline packages/cashier/src/pages/payment-agent/payment-agent-signup/steps/address-verification/proof-of-address-form/proof-of-address-form.scss:40]:
```suggestion
            height: 24rem;
```

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.scss (hunk 1 above)
Comment (hamid-deriv) [reply packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.scss:16]: Fixed.

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.scss (hunk 2 above)
Comment (hamid-deriv) [reply packages/cashier/src/pages/payment-agent/payment-agent-signup/signup-wizard.scss:32]: Fixed.

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/steps/address-verification/proof-of-address-form/proof-of-address-form.scss (hunk 6 above)
Comment (hamid-deriv) [reply packages/cashier/src/pages/payment-agent/payment-agent-signup/steps/address-verification/proof-of-address-form/proof-of-address-form.scss:17]: Sorted.

File: packages/cashier/src/pages/payment-agent/payment-agent-signup/steps/address-verification/proof-of-address-form/proof-of-address-form.scss (hunk 7 above)
Comment (hamid-deriv) [reply packages/cashier/src/pages/payment-agent/payment-agent-signup/steps/address-verification/proof-of-address-form/proof-of-address-form.scss:40]: Sorted.

**Review by farzin-deriv — APPROVED** [review]

**Review by george-usynin-binary — APPROVED** [review]

**Pull request merged**